src/renderer/containers/main/window.ts

```diff
@@ -12,6 +12,7 @@ class Window extends Container<WindowState, MainCTX> {
 
   state = {
     focus: false,
+    statusbar: false,
```

Comment:
This isn't the right container for this state, you should move the code you added from this file to the "editor" container.